.scalafmt.conf

```diff
@@ -1,9 +1,52 @@
+version = 2.1.0
+
 style = defaultWithAlign
 
-align.tokens               = [off]
-danglingParentheses        = true
 docstrings                 = JavaDoc
 indentOperator             = spray
 maxColumn                  = 120
-rewrite.rules              = [RedundantBraces, RedundantParens, SortImports]
+rewrite.rules              = [RedundantParens, SortImports, AvoidInfix]
 unindentTopLevelOperators  = true
+align.tokens               = [{code = "=>", owner = "Case"}]
+align.openParenDefnSite    = false
+align.openParenCallSite    = false
+optIn.breakChainOnFirstMethodDot = false
+optIn.configStyleArguments = false
```

Comment:
(I'm thinking we should drop this one, but I understand you just followed akka's config here, so I'll raise it for discussion there ;) )

Reply:
yep, copy-n-pase. Glad to adapt with we change everywhere